corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -1099,10 +1105,24 @@ hqDefine("cloudcare/js/form_entry/entries", [
         };
 
         self.resizeCanvas = function () {
-            var aspectRatio = 4,
-                width = self.$wrapper.width() - 2; // otherwise misaligned by 2px
+            const aspectRatio = 4;
+            const width = self.$wrapper.width() - 2; // otherwise misaligned by 2px
+            const scale = width / self.widthAtDataCapture;
             self.$canvas[0].width = width;
             self.$canvas[0].height = width / aspectRatio;
+
+            if (self.signatureData) {
+                const scaledData = JSON.parse(JSON.stringify(self.signatureData));
+                for (let i = 0; i < scaledData.length; i++) {
+                    const stroke = scaledData[i];
+                    for (let j = 0; j < stroke.points.length; j++) {
+                        const point = stroke.points[j];
+                        point.x = point.x * scale;
+                        point.y = point.y * scale;
+                    }
+                }
```

Comment:
Is there a reason not to modify `self.signatureData` directly? Since we have underscore available here, could this be simplified to something like:
```javascript
            _.each(data, stroke => {
                _.each(stroke.points, point => {
                    point.x *= scale;
                    point.y *= scale;
                })
            })
```

Reply:
I still think it'd be nicer to avoid the old school for loops here (replace `data` with `scaledData` in my snippet), but that's just personal preference.